shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java

```diff
@@ -374,15 +374,17 @@ private void setSystemChromeSystemUIOverlayStyle(
     // If transparent, SDK 29 and higher may apply a translucent scrim behind the bar to ensure
     // proper contrast. This can be overridden with
     // SystemChromeStyle.systemStatusBarContrastEnforced.
-    if (systemChromeStyle.statusBarIconBrightness != null && Build.VERSION.SDK_INT >= 23) {
-      switch (systemChromeStyle.statusBarIconBrightness) {
-        case DARK:
-          // View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
-          flags |= 0x2000;
-          break;
-        case LIGHT:
-          flags &= ~0x2000;
-          break;
+    if (Build.VERSION.SDK_INT >= 23) {
+      if (systemChromeStyle.statusBarIconBrightness != null) {
+        switch (systemChromeStyle.statusBarIconBrightness) {
+          case DARK:
+            // View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
```

Comment:
This value was [deprecated](https://developer.android.com/reference/android/view/View#SYSTEM_UI_FLAG_LIGHT_STATUS_BAR) in API 30.   

Do we know if this still works in Build.VERSION.SDK_INT >= 30 ?

Reply:
Yeah, the migration should probably be in a separate change though so all of this file can be migrated.